### DISCUSSION TITLE
pytest not working

### DISCUSSION THREAD

**Discussion by Ajinkya-9561** [pytest-dev/pytest#10135] [General]
![line 1](https://user-images.githubusercontent.com/109277916/178946774-2bb0a3fb-ef8c-4b6c-8d83-5089c8e9b242.png)
pytest in not working in pycharm
![line 1](https://user-images.githubusercontent.com/109277916/178946667-dc546785-3726-43ee-a06d-c0eaa2a3deab.png)

**Comment by RonnyPfannschmidt** [comment]
at first glance this looks like your virtualenv configuration and your powershell configuration are not set up correctly, this is not a pytest issue

this is a dev env on windows issue, please follow a tutorial for your platform (i haven't used windows in over a decade)

**Reply by Ajinkya-9561** [reply]
@Ronny Thank you. I'll do this.

**Reply by ronny** [reply]
@Ajinkya-9561 wrong person
